components/email/src/Email.svelte

```diff
@@ -587,22 +589,20 @@
     const existingDraft = activeThread?.drafts?.find(
       (draft) => draft.reply_to_message_id === message.id,
     );
+    const subject = `Fwd: ${message.subject}`;
     if (existingDraft) {
-      dispatchDraftEvent(event, existingDraft);
+      existingDraft.subject = subject;
+      dispatchEvent("forwardClicked", {
+        event,
+        message: existingDraft,
+        thread: activeThread,
+      });
     } else {
       //Create new message
```

Comment:
@AaronDDM  No longer need to include the `value` in `event.detail` since the properties and values are already in `event.detail.message`.

Reply:
@AaronDDM I've added them to the CHANGELOG, do you think it'll be better to keep `event.detail.value` to preserve the backward compatibility?

Reply:
Yep, I don't think we need to introduce a BC chagne just yet.